admin/client.py

```diff
@@ -1,176 +1,164 @@
 # Copyright Hybrid Logic Ltd.  See LICENSE file for details.
 """
-Run the acceptance tests.
+Run the client installation tests.
 """
 
+import os
+import shutil
 import sys
+import tempfile
 import yaml
 
-from zope.interface import Interface, implementer
-from characteristic import attributes
-from eliot import add_destination
-from twisted.internet.error import ProcessTerminated
+import docker
+from effect import TypeDispatcher, sync_performer, perform
 from twisted.python.usage import Options, UsageError
 from twisted.python.filepath import FilePath
-from twisted.internet.defer import inlineCallbacks, returnValue
 
-from flocker.common.version import make_rpm_version
-from flocker.provision import PackageSource, CLOUD_PROVIDERS
 import flocker
-from flocker.provision._ssh import (
-    run_remotely)
+from flocker.common.version import make_rpm_version
+from flocker.provision import PackageSource
+from flocker.provision._effect import Sequence, perform_sequence
```

Comment:
If Sequence is replaced for docs and cloud tests, it won't take much further effort to change it here.  It's not clear how long it will take for FLOC-2810 to be started and accepted, so I'd rather proceed using the current methods here.